test/parallel/test-http-writable-true-after-close.js

```diff
@@ -12,18 +12,18 @@ let external;
 // Http server
 const internal = createServer((req, res) => {
   res.writeHead(200);
-  setImmediate(common.mustCall(() => {
+  setTimeout(common.mustCall(() => {
     external.abort();
     res.end('Hello World\n');
-  }));
+  }), common.platformTimeout(100));
 }).listen(0);
 
 // Proxy server
 const server = createServer(common.mustCall((req, res) => {
+  res.on('close', common.mustCall(() => {
+    assert.strictEqual(res.writable, true);
+  }));
   get(`http://127.0.0.1:${internal.address().port}`, common.mustCall((inner) => {
-    res.on('close', common.mustCall(() => {
-      assert.strictEqual(res.writable, true);
-    }));
     inner.pipe(res);
```

Comment:
Is it safe to call `external.abort()` and `internalRes.end('Hello World\n');` here or does this invalidate the test?
This would remove the `setTimeout` dependency.